mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/scheduler/CpuWeightedFitnessCalculator.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2024 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.mantisrx.master.scheduler;
+
+import io.mantisrx.runtime.MachineDefinition;
+
+/**
+ * Implementation of FitnessCalculator. Uses cpu cores with higher weight than memory.
+ */
+public class CpuWeightedFitnessCalculator implements FitnessCalculator {
+    @Override
+    public double calculate(MachineDefinition requested, MachineDefinition available) {
+        if (!available.canFit(requested)) {
```

Comment:
GIven we don't want to handle anything for disk/bandwidth/ports maybe let's remove the old check? (also I am thinking the case where we might want to adjust network bw more dynamically).

Reply:
"we don't want to handle anything for disk/bandwidth/ports" I don't think that's totally accurate, or at least this wasn't my intent. My intent was to still verify requested disk/bw/port fit in available containers and that's the reason I kept the `canFit` call. I'm not sure we want to change that.